en/guides/students/login-and-settings/index.md

```diff
@@ -1,199 +1,79 @@
+You can use Dodona in English or Dutch. Use the language menu on the right side of the navigation bar (at the top of every page) for this purpose.
 
 ![image](./choose_language.png)
 
-Als je een taal selecteert terwijl je aangemeld bent dan wordt die ook als voorkeurstaal ingesteld in je [gebruikersprofiel](https://dodona.be/profile). Op die manier wordt bij aanmelden automatisch je voorkeurstaal geselecteerd.
+If you select a language while logged in, it will also be set as your preferred language in your [user profile](https://dodona.be/profile). This way, your preferred language will be automatically selected upon login.
 
-::: tip Wordt alles vertaald?
+::: tip Is Everything Translated?
 
-De vaste componenten van Dodona (menubalken, cursusstructuur, mededelingen, ...) worden altijd weergegeven in de geselecteerde taal.
+The fixed components of Dodona (menu bars, course structure, notifications, etc.) are always displayed in the selected language.
 
-De naam en beschrijving van een [cursus](../courses/) worden weergegeven in de taal zoals de lesgever ze heeft ingegeven. Deze worden niet aangepast aan je taalvoorkeuren.
+The name and description of a [course](../courses/) are displayed in the language as entered by the instructor. These are not adapted to your language preferences.
 
-De naam en beschrijving van een [oefening](../exercises/#navigeren-naar-een-oefening) worden steeds in je voorkeurstaal weergegeven indien beschikbaar. Niet alle oefeningen zijn namelijk in beide talen beschikbaar.
+The name and description of an [exercise](../exercises/#navigating-to-an-exercise) are always displayed in your preferred language if available. Not all exercises are available in both languages.
 
 :::
 
-## Aanmelden
+## Signing Up
 
-Om je aan te melden op dodona klik je ofwel op `Aanmelden` aan de rechterkant van de navigatiebalk, ofwel navigeer je naar de landingspagina en klik je op de knop `Aanmelden` centraal op de pagina.
+To sign up for Dodona, either click on `Sign in` on the right side of the navigation bar or navigate to the landing page and click the `Sign in` button in the center of the page.
 
 ![image](./login.png)
 
-Je kan je aanmelden op Dodona door je onderwijsinstelling te selecteren op de aanmeldpagina. Studenten uit het secundair kunnen inloggen met hun Smartschool, Office365 of G-Suite account. Als dit de eerste keer is dat je aanmeldt op Dodona dan wordt er automatisch een nieuwe gebruikersaccount aangemaakt op basis van de persoonsgegevens die doorgestuurd werden door je externe onderwijs- of onderzoeksinstelling.
+You can sign up for Dodona by selecting your educational institution on the sign-up page. Secondary school students can log in with their Smartschool, Office365, or G-Suite account. If this is your first time signing up for Dodona, a new user account will be automatically created based on the personal information provided by your external educational or research institution.
```

Comment:
```suggestion
You can sign in to Dodona by selecting your educational institution on the sign-in page. Secondary school students can log in with their Smartschool, Microsoft 365, or Google Workspace account. If this is your first time signing in to Dodona, a new user account will be automatically created based on the personal information provided by your external educational or research institution.
```
This might need a change in Dutch as well.